README.md

```diff
@@ -7,7 +7,10 @@ Logux is a new way to connect client and server. Instead of sending
 HTTP requests (e.g., AJAX and GraphQL) it synchronizes log of operations
 between client, server, and other clients.
 
-**Documentation: [logux.io]**
+* **[Guide and recipes](https://logux.io/)**
+* **[Chat](https://gitter.im/logux/logux)** for any questions
+* **[Issues and plans](https://github.com/logux/logux/issues)**
```

Comment:
For consistency, I suggest «Issues and roadmap».

Reply:
Fixed https://github.com/logux/vuex/pull/28/commits/a491e7d46cda4564b7827fa38b5d6ccf7ec543ee